docs/sponsors.md

```diff
@@ -2,5 +2,7 @@
 
 Sponsorship for the PyTexas Virtual Meetup's been wrangled up by these fine organizations below. We're as grateful as a longhorn in spring grass! A big Texas-sized THANK YOU to each one of 'em.
 
+(Want to see your company or organization on this list? Send a warm howdy to sponsorship@pytexas.org)
```

Comment:
I debated on also saying what sponsorship entails (e.g. a door prize). Thoughts on including that here?

Reply:
Could it not also entail helping out with primo-discord?

Reply:
Feel free to suggest some wording, otherwise :man_shrugging:

Reply:
I think it's good enough. I'll take whatever they want to give. If someone wanted to make a 1 pager for Meetup sponsorship and link to it here I'd be fine with that.